scripts/run.yaml

```diff
@@ -14,3 +14,8 @@ runs:
     command: /usr/bin/sqlite3 
     memory: 64
     networking: False
+  - name: bzip2
+    rootfs: ../dynamic-apps/bzip2
+    command: /bin/bzip2
```

Comment:
This command won't work. Add a proper running command.

Do the following:

- Add a new application to `run.yaml`.
- Generate the run script with `./scripts/generate.py`.
- Build `app-elfloader` with a build script of choice from `./scripts/build/...`
- Run the binary configured in `run.yaml` with the corresponding script from `./scripts/run/...`.

See instructions in [the "Scripted Building and Running" section in `README.md`](https://github.com/unikraft/app-elfloader#scripted-building-and-running).